package.json

```
@@ -15,6 +15,8 @@
     "build:lib:watch": "lerna run build:lib:watch --parallel",
     "build:types:watch": "lerna run build:types:watch --parallel",
     "build:watch": "run-p build:lib:watch build:types:watch",
+    "test:unit": "node_modules/.bin/lerna run test:unit --no-bail",
+    "test:e2e": "node_modules/.bin/lerna run test:e2e --no-bail",
```

```suggestion
    "test:e2e": "lerna run test:e2e --no-bail",
```
works inside package.json^^